### ISSUE TITLE
False positive with Layout/MultilineMethodCallIndentation on indented lines

### ISSUE THREAD

**Issue by sam-carlberg** [rubocop-hq/rubocop#7550]
Given this snippet:

```ruby
def example
  elements = [1, 2]
  elements.first.to_s +
    elements.second
      .to_s # Layout/MultilineMethodCallIndentation error - use 2 (not 4) spaces
end
```

Enforced style is set to `indented` in `.rubocop.yml`

##  Expected behavior

I expect this line to not be flagged, as it is properly indented relative to the line it's chained from

## Actual behavior

Rubocop has a false positive and autocorrects to this format:

```ruby
def example
  elements = [1, 2]
  elements.first.to_s +
    elements.second
    .to_s
end
```

## Steps to reproduce the problem

Run Rubocop on a file with the above code snippet.

## RuboCop version

```
0.77.0 (using Parser 2.6.5.0, running on ruby 2.6.5 x86_64-darwin18)
```

**Comment by AlexWayfer** [comment]
Looks related to #7338.

Also there is a case with `<<`, for example:

```ruby
elements <<
  new_element
    .to_s
```

**Comment by AlexWayfer** [comment]
Not sure is it correct to "bump" it.

**Comment by sam-carlberg** [comment]
This seems to be fixed as of 1.11.0. Running rubocop on the autocorrected snippet in the example I gave now warns use to 2 spaces of indentation, not 0.
